### DISCUSSION TITLE
Android 11 and building bundles

### DISCUSSION THREAD

**Discussion by phusting** [apache/cordova#376] [Q&A]
With the upgrade to Android 11, I guess we have to allow the Play Store to manage our keys now?  I can't use the key I generated 4+ years ago anymore?  Any suggestions before I start pouring through the android documents for doing this?  :)

**Comment by breautek** [comment]
You still use your key to sign your bundles. But you must enable google key management as well. This is because Google effectively builds out an APK on the fly for your users tailored to their device using the resources in the bundle (however hybrid apps don't really benefit much from this)

**Reply by phusting** [reply]
Okay, Is Google Key Management in the Play Console or is that the Development Studio?  Sorry, not an Android expert.

**Reply by breautek** [reply]
It's been awhile for me so some details here might not be fully accurate.... but if you go through the instructions on Google Play it's going to instruct you build a new key for Google play, using an existing key (the key you sign your APKs) . You upload the new key which Google uses and you build and sign your bundles using your existing key that you used to sign your APKs with.

Should be safe to follow the instructions in the Play Console though.

**Reply by phusting** [reply]
I got through it today....  I got a new warning message after uploading....  "**You have not completed your government declaration**"  This is kind of worrying...

**Reply by breautek** [reply]
It just means you haven't filled out the App Content forms completely yet. I believe there is a single question that asks if the app is for government use or not, or something to that effect. Under `Policy and programs` (Last menu section) , go to `App Content`.

Any required forms that you haven't completed will be on the top, there will be one for `Government apps`.
